internal/config/config.go

```diff
@@ -34,6 +40,7 @@ type SauceConfig struct {
 	Experiments map[string]string `yaml:"experiments,omitempty" json:"experiments,omitempty"`
 	Retries     int               `yaml:"retries,omitempty" json:"-"`
 	Visibility  string            `yaml:"visibility,omitempty" json:"-"`
+	LaunchBy    LaunchBy          `yaml:"launchBy,omitempty" json:"launchBy,omitempty"`
```

Comment:
Despite its brevity, I do wonder if `launchOrder` is more self explanatory.

Reply:
ok